src/xhr.js

```diff
@@ -1,5 +1,19 @@
 import { Headers as HeadersPolyfill } from './polyfill/Headers';
 
+function getAbortError() {
```

Comment:
Nit pick: as this constructs a new instance could we please prefix using make or create instead of get which, IMHO is ambiguous in this regard